recipes/ayufan-release-contents.md

```diff
@@ -0,0 +1,151 @@
+./usr/share/alsa/init/rt5616
+./usr/share/alsa/init/rt5640
+./usr/share/alsa/init/rt5651
+./usr/share/backgrounds/ubuntu-mate-rock64/ROCK64-Wallpaper-1.jpg
+./usr/share/backgrounds/ubuntu-mate-rock64/ROCK64-Wallpaper-2.jpg
+./usr/share/backgrounds/ubuntu-mate-rock64/ROCK64-Wallpaper-3.jpg
+./usr/share/backgrounds/ubuntu-mate-rock64/ROCK64-Wallpaper-4.jpg
+./usr/share/backgrounds/ubuntu-mate-rock64/ROCK64-Wallpaper-5.jpg
+./usr/share/backgrounds/ubuntu-mate-rock64/ROCK64-Wallpaper-6.jpg
+./usr/share/doc/linux-rock64-package/changelog.gz
+./usr/share/flash-kernel/db/rock64.db
+```
+
+Most files are there to ensure the hardware features of the `rock64` work well. Some scripts can also help with debugging or improving your `rock64` experience:
+
+  * `rock64_health.sh`: outputs the CPU frequency, count, and SoC temperature
+  * `rock64_eth0_stats.sh`: outputs statistics about your `eth0` network adapter
+  * `install_desktop.sh`: installs a Linux desktop packages if you've installed a `minimal` distribution
+
+It's worth noting, many of these customizations have been borrowed from `pine64`, `armbian`, and other tools designed for SBCs.
+
+## How are they built
+
+A publicly accessible [Jenkins server](https://jenkins.ayufan.eu/job/linux-build-rock-64/) is used to automate creation of various builds. The [full log](https://jenkins.ayufan.eu/job/linux-build-rock-64/lastBuild/console) is available for anyone to see.
+
+The aptly named [linux-build](https://github.com/ayufan-rock64/linux-build) repository contains Makefiles and build scripts for generating each build.
+
+## Can they be trusted
+
+Well, as much as you trust the official Debian or Ubuntu distributions.
+
+A lot of work has been made to ensure visibility into the entire build process, which can't be said for all Linux distributions out there.
+
+You're always free to inspect the sources, build logs, and compare outputs with other similar systems.
+
+## But I want my own
+
+Since all build scripts and sources are available, it is possible to create your own distribution to run on the `rock64`. Many people have done so with systems such as Devuan and RancherOS.
+
+We won't go into details on _how_ to do it, but it's definitely possible if you're knowledgeable with Linux.
+
+## Who is ayufan
+
+Just some [guy](https://github.com/ayufan). [Buy him a beer](https://www.paypal.me/ayufanpl), he deserves it.
+
+## Are there alternatives
+
+There are many options for the `rock64`, depending on your needs. It is possible to run Android, Armbian, Yocto, and a host of other `arm64` systems on the `rock64`.
+
+A comprehensive list of available operating systems are found here:
+
+  * [http://wiki.pine64.org/index.php/ROCK64_Software_Release](http://wiki.pine64.org/index.php/ROCK64_Software_Release)
+  * [https://www.armbian.com/rock64/](https://www.armbian.com/rock64/)
+  * [https://github.com/ayufan-rock64/linux-build/releases](https://github.com/ayufan-rock64/linux-build/releases)
+
+## FAQ
```

Comment:
Can we advise to use issues and search of this repository to provide FAQs?

Reply:
maybe the FAQ section is redundant. I'll remove it completely.